hibernate-core/src/main/java/org/hibernate/boot/model/source/internal/hbm/AuxiliaryDatabaseObjectBinder.java

```diff
@@ -68,4 +73,51 @@ public static void processAuxiliaryDatabaseObject(
 
 		context.getMetadataCollector().getDatabase().addAuxiliaryDatabaseObject( auxDbObject );
 	}
+
```

Comment:
I think we should move this to a dedicated binder related to mapping.xml.  Ultimately we will drop hbm.xml processing and woould be better at that point to not have to sift through hbm.xm-processing code to pull out bits related to mapping.xml.  Does that make sense?